vision_evaluation/evaluators.py

```diff
@@ -1100,3 +1100,46 @@ def add_predictions(self, predictions, ground_truths):
 
     def get_report(self, **kwargs):
         return {self._get_id(): {group: eval.get_report(**kwargs) for group, eval in self._evaluators.items()}}
+
+
+class MeanLpErrorEvaluator(Evaluator):
+    """
+    Error evaluator for regression targets
+    Parameters:
+        p: Lp order of the norm used
+    """
+
+    def __init__(self, p: int):
+        assert p != 0
+        self.p = p
+
+        self.total_num = 0
+        self.total_error = 0
+
+        super(MeanLpErrorEvaluator, self).__init__()
+        
+    def reset(self):
+        super(MeanLpErrorEvaluator, self).reset()
+        self.total_num = 0
+        self.total_error = 0
+
+    def _get_id(self):
+        return f'l{self.p}'
```

Comment:
return f'MeanL{self.p}Error'

Reply:
Thanks! Modified slightly to follow the format of the other metrics; I used lowercase + underscores.